site/app/templates/polls/PollForm.twig

```diff
@@ -202,6 +221,36 @@
         }
         submitErrorChecks();
     };
+    window.onload = function() {
+        toggleTimerInputs();
+    };
+    function toggleTimerInputs() {
+        var enableTimerCheckbox = document.getElementById("enable-timer");
+        var timerInputs = document.getElementById("timer-inputs").querySelectorAll(".poll-timer");
+
+        if (enableTimerCheckbox.checked) {
+            document.getElementById("timer-inputs").style.display = "block";
+            timerInputs.forEach(function(input) {
+                input.setAttribute("type", "number");
+                input.setAttribute("min", "0");
+                if (input.classList.contains("poll-hours")) {
+                    input.setAttribute("placeholder", "Hour");
+                    input.setAttribute("max", "24");
+                } else if (input.classList.contains("poll-minutes")) {
+                    input.setAttribute("placeholder", "Min");
+                } else if (input.classList.contains("poll-seconds")) {
+                    input.setAttribute("placeholder", "Sec");
+                }
+            });
+        } else {
+            document.getElementById("timer-inputs").style.display = "none";
```

Comment:
I believe you can use `.hide()` instead of manipulating the style directly.

Reply:
I changed the function to use jquery library.